ckanext/xloader/parser.py

```diff
@@ -1,161 +1,71 @@
 # -*- coding: utf-8 -*-
-import csv
+import datetime
 from decimal import Decimal, InvalidOperation
-from itertools import chain
+import re
+import six
 
 from ckan.plugins.toolkit import asbool
-from dateutil.parser import isoparser, parser
-from dateutil.parser import ParserError
-
-from tabulator import helpers
```

Comment:
what else was in the parser we don't have now.

Reply:
I did a diff between our parser and the vanilla one. Adding the conversion to numbers and dates was it.